dev-requirements.txt

```diff
@@ -11,3 +11,4 @@ mock
 # these are optional for users, but for testing we need them
 ipaddr
 GeoIP
+stem
```

Comment:
At this point, perhaps we should add Tox environments that have (or do not have) the optional req's....but, see comment below.